helpers/lib/fonts.js

```diff
@@ -84,54 +88,76 @@ const fontProviders = {
  * @returns {Object.<string, Array>|string}
  */
 module.exports = function(format, themeSettings, handlebars, options) {
-    
+
     const collectedFonts = {};
-    _.each(themeSettings, function(value, key) {
+    const themeSettingsKeys = Object.keys(themeSettings);
+
+    for (let i = 0; i < themeSettingsKeys.length; i++) {
+        const key = themeSettingsKeys[i];
+        const value = themeSettings[key];
         //check that -font is on end of string but not start of string
         const fontKeySuffix = '-font';
         if (!key.endsWith(fontKeySuffix)) {
-            return;
+            continue;
         }
 
         const splits = value.split('_');
         const provider = splits[0];
 
         if (typeof fontProviders[provider] === 'undefined') {
-            return;
+            continue;
```

Comment:
Are we ok to change return to continue here?

Reply:
yes, because return in a forEach is equivalent to continue in a for